scripts/fast-reboot

```diff
@@ -407,12 +407,13 @@ docker kill lldp > /dev/null
 systemctl stop lldp
 
 if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
+    # Kill teamd processes inside of teamd container with SIGINT to allow them to send last LACP frames
     # Kill teamd, otherwise it gets down all LAGs
     # We call `docker kill teamd` to ensure the container stops as quickly as possible,
     # then immediately call `systemctl stop teamd` to prevent the service from
     # restarting the container automatically.
     # Note: teamd must be killed before syncd, because it will send the last packet through CPU port
-    # TODO: stop teamd gracefully to allow teamd to send last valid update to be sure we'll have 90 seconds reboot time
+    docker exec -i teamd pkill -INT teamd
     docker kill teamd > /dev/null
```

Comment:
Is there no need for a brief sleep (or wait for all teamd processes to exit) after sending the SIGINT and before killing the container? Can we guarantee the teamd processes were all able to receive the signal and send the last LACP frames?

Reply:
I have the same thought. We can't guarantee that. You're right. But every sleep increase pause of control plane. Also I'm not sure which sleep time is good enough. I feel that 1 sec will work good for 99% cases, but I can't prove it.

Reply:
I think the process should have been killed when kill command returns.

Reply:
I'll write a loop to check that.

Reply:
@yxieca: `kill` simply sends a signal to a process. It has no way of knowing whether the signal will cause the process to terminate. Thus, kill returns after it sends the signal; it doesn't wait for anything. A loop to check for the existence of the process(es) like Pavel just added is the best method of ensuring we don't kill the container prematurely.